src/main/java/roomescape/controller/ReservationTimeController.java

```diff
@@ -0,0 +1,46 @@
+package roomescape.controller;
+
+import java.net.URI;
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.util.UriComponentsBuilder;
+import roomescape.dto.ReservationTimeReqDto;
+import roomescape.dto.ReservationTimeResDto;
+import roomescape.service.ReservationTimeService;
+
+@RestController
+@RequestMapping("/times")
+public class ReservationTimeController {
+
+    private final ReservationTimeService reservationTimeService;
+
+    public ReservationTimeController(ReservationTimeService reservationTimeService) {
+        this.reservationTimeService = reservationTimeService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<ReservationTimeResDto>> readAll() {
+        List<ReservationTimeResDto> response = reservationTimeService.findAll();
+        return ResponseEntity.ok(response);
```

Comment:
변수 초기화 없이도 할 수 있곘네요! :)

```suggestion
        return ResponseEntity.ok(reservationTimeService.findAll());
```

Reply:
언제 변수 초기화를 고려해도 괜찮을지 항상 고민되는 것 같습니다! `reservationTimeService.findAll()` 메서드 호출은 변수에 따로 할당하지 않고 그 반환값을 바로 매개변수로 전달할 만큼 복잡하지 않다고 여겨지긴 합니다. 👀 만약 스트림 API 사용처럼 메서드 체이닝이 있는 경우엔 따로 변수에 저장할 것 같은데, 혹시 찰리는 특별히 선호하시는 스타일이나 기준이 있으신지 궁금합니다!

Reply:
[refactor: 메서드 호출을 변수 저장 없이 매개변수로 전달](https://github.com/woowacourse/spring-roomescape-admin/pull/359/commits/f96d53de8fe4cd28892457b6907720a92c07640d)

Reply:
대부분의 경우 변수 할당은 지양하는게 좋다고 생각해요!
변수를 할당하면 그만큼 GC 에도 영향을 줄 수 있으니까요
요즘은 성능이 좋아서 걱정해야하냐 할 수 있는데
굳이 불필요한 변수 할당을 하지말자 라는 취지죠 ㅎㅎ

가독성을 위한 변수 할당은
변수 할당을 하지않으면 이해하기 어렵겠다
라는 상황일 때 변수 할당하는것을 고려하고 있어요

그외에 꼭 필요한 변수 할당은 당연하다고 생각합니다!
재사용을 위해서라던지
흐름에 필요하다던지